README.md

```diff
@@ -30,7 +30,6 @@ yarn create wagmi --template next-with-connectkit
 - `next-with-connectkit`: A Next.js wagmi project with ConnectKit included.
 - `next-with-rainbowkit`: A Next.js wagmi project with RainbowKit included.
 - `vite-react`: A Vite (React) wagmi project.
```

Comment:
We should update the list in the README to reflect the updated template structure (e.g. Next/Vite x ConnectKit/RainbowKit/Web3Modal)